### PR TITLE
Update Muon PhaseQuad documentation and user examples

### PR DESCRIPTION
**Pull request by raquelalvarezbanos**
Fixes [#11191](http://trac.mantidproject.org/mantid/ticket/11191)

To test: this ticket is to update the algorithm's documentation, so basically the only two things to test here are the user examples.

### CODE REVIEW

File: Code/Mantid/docs/source/algorithms/PhaseQuad-v1.rst [hunk 1]
```diff
@@ -35,12 +35,12 @@ Usage
...
-.. testcode:: ExCompSquash
+.. testcode:: ExPhaseQuadList
 
-   # Load the first two spectra from a MUSR run
-   input = LoadMuonNexus('MUSR0015189.nxs',EntryNumber=1,SpectrumMin=1,SpectrumMax=2)
+   # Load a set of spectra from a EMU file
+   ws = LoadMuonNexus('EMU00006473.nxs')
 
    # Create a PhaseList file with some arbitrary detector information
    file = open('PhaseList.txt','w')
```
Comment (martyngigg) [inline Code/Mantid/docs/source/algorithms/PhaseQuad-v1.rst]: There is nothing wrong with the line here, however, on some users' machines the current working directory is not writeable so this example may not work out of the box. In most of our examples that write files we tend to use the home directory, which is easy to access in Python. Can we change this line to something like
```python
phaselist_path = os.path.join(os.path.expanduser("~"),"PhaseList.txt")
phaselist_file = open(phaselist_path, 'w')
```
Note that I have also replaced `file` with`phaselist_file` as `file` is a keyword in Python and is generally considered bad practice to over write keywords.

**Comment by martyngigg** [comment]
Could we also remove the temporary `PhaseList` file after we are finished? This just avoids having random files scattered around the build directories. This can be done outside of the test by adding the following lines underneath the `.. testoutput:: ExPhaseQuadList` section:

```python
.. testcleanup:: ExPhaseQuadList

  import os
  try:
      os.remove(phaselist_path)
  except OSError:
      pass
```

See `CreateDummyCalFile-v1.rst` for an example.

**Comment by raquelalvarezbanos** [comment]
Sure, I will remove the file. Thanks Martyn

**Comment by raquelalvarezbanos** [comment]
I have just seen I missed the first comment, I will modify that too.

**Comment by raquelalvarezbanos** [comment]
Jenkins, retest this please

**Comment by martyngigg** [comment]
It looks like this doesn't merge cleanly with master any more. Could you merge master to your branch?

**Comment by martyngigg** [comment]
Jenkins, retest this please

**Comment by raquelalvarezbanos** [comment]
Jenkins, retest this please

**Comment by martyngigg** [comment]
One of the builds has failed with a git error. I don't think it is worth pushing this through the build servers again. I'll just wait until the build finish and merge it then.

**Comment by martyngigg** [comment]
The `rhel6` failure is nothing to do with the changes here. The documentation tests have passed and rhel6 built before.

**Pull request merged**